requirements-dev.txt

```diff
@@ -28,3 +28,4 @@ cloudpickle
 rpyc==4.1.5
 asv
 xgboost>=1.3
+tqdm
```

Comment:
Sorry for the late comment but still why was `tqdm` added into `requirements-dev.txt` file only?

Reply:
cc @noah-kuo , @devin-petersohn

Reply:
We can add it to the other development environment files. It's mostly for the manual testing since we don't have automatic testing for the visualization of the bar. 

If you'd like to add it to the other files, please feel free!

Reply:
I just wanted to make sure that is not intentionally missed. We should add it to the other development environment file in order to keep the files up to date. We could add it as a part of the #2775 or make a separate PR for it.

Reply:
Let's make it a separate PR, but yes let's add it. Will you be able to do it?

Reply:
Yes, I will create a PR for it.